source/content/object-cache.md

```diff
@@ -35,7 +31,7 @@ All plans except for the Basic plan can use Redis. Sandbox site plans can enable
 
 <Tab title="WordPress" id="wp-install" active={true}>
 
-1. Enable Redis from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**. It may take a couple minutes for the Redis server to come online.
+1. Enable Object Cache from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**. It may take a couple minutes for the Object Cache server to come online.
 
 1. Install the [WP Redis](https://wordpress.org/plugins/wp-redis/) plugin via SFTP or Git. To install via [Terminus](/terminus), [set the connection mode to SFTP](/sftp) then run:
```

Comment:
wp-redis plugin is probably not going to be renamed

Reply:
This has been logged as a separate potential task in Jira, outside the scope of this PR.